go/api/base_client.go

```diff
@@ -7011,3 +7011,42 @@ func (client *baseClient) Time() ([]string, error) {
 	}
 	return handleStringArrayResponse(result)
 }
+
+// Calculates the difference between the first sorted set and all the successive sorted sets at
+// `keys` and stores the difference as a sorted set to `destination`,
+// overwriting it if it already exists. Non-existent keys are treated as empty sets.
+//
+// Note: When in cluster mode, `destination` and all `keys` must map to the same hash slot.
+// Available for Valkey 6.2 and above.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//  destination - The key for the resulting sorted set.
+//  keys        - The keys of the sorted sets to compare.
+
```

Comment:
```suggestion
//
```
Use blank line or "//"?

Reply:
Updated.